### ISSUE TITLE
False positive for "Unnecessary semicolon"

### ISSUE THREAD

**Issue by mpceiv** [kalessil/phpinspectionsea#1649]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) v4.0.6 |
| Language level | PHP 7.3                        |

Current behaviour
--

```php
    $arr['something'] = 'Something';
    ?>
    <div>
        <span>
            <?= strtolower('TOLOWER'); echo ' ' . $arr['something'] ?>
        </span>
    </div>
```

Code is meaningless (and not mine) just to reproduce the bug. 

Expected behaviour
--

No warning for the short echo tag if there is another statement after.

Environment details
---

![image](https://user-images.githubusercontent.com/45010533/96880995-b5dfbc00-1475-11eb-9aad-ba2bccb303fe.png)

**Comment by ea-inspections-team** [comment]
Thank you for reporting @mpceiv! Fixed!
